Collector/ProfilePlugin.php

```diff
@@ -58,7 +58,9 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
     {
         $profile = new Profile($this->pluginName, $this->formatter->formatRequest($request));
 
-        $this->collector->getCurrentStack()->addProfile($profile);
+        if (false !== $stack = $this->collector->getCurrentStack()) {
```

Comment:
i would write it like jerome proposes, thats simpler to read.

the question remains if this situation can ever happen. maybe we should instead have getCurrentStack throw an exception if there is no current stack. that is more disruptive but would force to figure out why it happens.

though maybe as this is just a debugging tool, we should rather silently fail than make a noise about it.